migrate/20241213_archived_record.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:archived_record, partition_by: :deleted_at, partition_type: :range) do
```

Comment:
Maybe use `:archived_at` as column name to match with the table name?

Reply:
I don't have a strong opinion on this. I felt that `deleted_at` still works, like soft deletion of a record puts it into the archive with a deleted_at timestamp on it. But `archived_at` might be more consistent. I can change the name.